salt/orchestrate/edx/init.sls

```diff
@@ -13,7 +13,9 @@ generate_edx_cloud_map_file:
     - template: jinja
     - makedirs: True
     - context:
+        business_unit: {{ BUSINESS_UNIT }}
         environment_name: {{ ENVIRONMENT }}
+        purpose: {{ PURPOSE }}
```

Comment:
I would propagate the `_prefix` nomenclature throughout so that it's easier to read and understand what it is intended to represent.